portfolio/src/main/webapp/script.js

```diff
@@ -70,4 +70,30 @@ function deleteComments() {
   fetch('/delete-comments');
   const commentElement = document.getElementById("comment-container");
   commentElement.innerHTML= "";
+}
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Fetches book data and uses it to create a chart. */
+function drawChart() {
+  fetch('/book-data').then(response => response.json())
+  .then((bookVotes) => {
+    const data = new google.visualization.DataTable();
+    data.addColumn('string', 'Book');
+    data.addColumn('number', 'Votes');
+    Object.keys(bookVotes).forEach((book) => {
+      data.addRow([book, bookVotes[book]]);
+    });
+
+    const options = {
```

Comment:
Does this really need to be assigned to a variable? Can't you just pass the constant dictionary directly in the call to chart.draw below?